{
  "Header": {
    "Original": "CVE-2025-39913 (In the Linux kernel, the following vulnerability has been resolved:  t ...)",
    "ID": "CVE-2025-39913",
    "Description": "(In the Linux kernel, the following vulnerability has been resolved:  t ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-4328-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-4328-1"
      ]
    },
    {
      "Original": "- linux 6.16.8-1",
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.16.8-1"
    },
    {
      "Original": "[trixie] - linux 6.12.48-1",
      "Type": "package",
      "Release": "trixie",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.12.48-1"
    },
    {
      "Original": "[bookworm] - linux 6.1.153-1",
      "Type": "package",
      "Release": "bookworm",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.1.153-1"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/a3967baad4d533dc254c31e0d221e51c8d223d58 (6.17-rc6)",
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/a3967baad4d533dc254c31e0d221e51c8d223d58 (6.17-rc6)"
    }
  ]
}